modules/islandora_collection_permissions/includes/handler.class.inc

```diff
@@ -155,7 +155,9 @@ class IslandoraCollectionPermissionsHandler {
           break;
         }
 
-        $this->pidsToExamine = array_merge($this->pidsToExamine, $this->getRelatedPids($operand_in));
+        if (is_array($operand_in) && count($operand_in)) {
+          $this->pidsToExamine = array_merge($this->pidsToExamine, $this->getRelatedPids($operand_in));
```

Comment:
This shouldn't work: If `$operand_in` was an array (and passed as such), the `::getRelatedPids()` bit should fall apart, since [it expects a PID to be passed as a string](https://github.com/discoverygarden/islandora_access_override/pull/3/files#diff-ddd23919e1bc4770f0b58bc9cf0e8696R172)...